edxval/api.py

```diff
@@ -148,7 +149,9 @@ def get_course_video_image_url(course_id=None, edx_video_id=None, video_image=No
     """
     try:
         if video_image is None:
-            video_image = CourseVideo.objects.get(course_id=course_id, video__edx_video_id=edx_video_id).video_image
+            video_image = CourseVideo.objects.select_related('video_image').get(
+                course_id=course_id, video__edx_video_id=edx_video_id
+            ).video_image
```

Comment:
In either case i think we should be specific about DoesNotExist based on it's model.